src/index.ts

```diff
@@ -160,7 +161,8 @@ export interface Options {
   cert?: string,
   githubToken?: string,
   webhookProxy?: string,
-  port?: number
+  port?: number,
+  redisConfig?: object | string
```

Comment:
Do you know if there is a type we can pull in here for this instead of just using `object`? Ideally we'd have a type that defines all the options of the [`ioredis` constructor](https://github.com/luin/ioredis/blob/0985ae8a9f057328b5f73dde708aa951b1713d9a/lib/redis.js#L21), I think.

Reply:
Great question @tcbyrd. I don't use typescript, but I looked into it. It looks like ioredis is in the process of converting to typescript (https://github.com/luin/ioredis/pull/794), but they have not yet created or exported the type that we'd need to import.

I also tried using `@types/ioredis`, and with a minor refactor I was able to get that working. Would you like me to update the PR to use @types/ioredis?

It seems to be that the best scenario is is ioredis ships type definitions directly, but it's probably worthwhile to use `@types/ioredis` while we wait?

Reply:
> but it's probably worthwhile to use `@types/ioredis` while we wait?

Yeh that seems fine for now, as long as the type definition is up to date. So I think we just need [`interface RedisOptions`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0b417416fe69ce507a5224f32924da296667b1fe/types/ioredis/index.d.ts#L875)?

Reply:
Yes, exactly. I've pushed a new commit with the required changes to make this work. Please review. I kept the commit separate so you can see the changes made but I'm happy to squash the commits if you'd like.

Reply:
I just pushed again to fix a lint failure.